internal/restore/volumesnapshotclass_restorer.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package restore
+
+import (
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	snapshotv1beta1api "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
+)
+
+// VolumeSnapshotClassRestoreItemAction is a Velero restore item action plugin for VolumeSnapshotClass
+type VolumeSnapshotClassRestoreItemAction struct {
+	Log logrus.FieldLogger
+}
+
+// AppliesTo returns information indicating that VolumeSnapshotClassRestoreItemAction should be invoked while restoring
+// volumesnapshotclass.snapshot.storage.k8s.io resrouces.
+func (p *VolumeSnapshotClassRestoreItemAction) AppliesTo() (velero.ResourceSelector, error) {
+	return velero.ResourceSelector{
+		IncludedResources: []string{"volumesnapshotclass.snapshot.storage.k8s.io"},
+	}, nil
+}
+
+// Execute restores volumesnapshotclass objects returning any snapshotlister secret as additional items to restore
```

Comment:
Minor nit: Period at the end for valid godoc string.

Reply:
added the `.` in https://github.com/vmware-tanzu/velero-plugin-for-csi/pull/37/commits/e4018c28326f338f6c346f6e8d40502d8c66ae7b